### DISCUSSION TITLE
FoxGlove Windows ROS2 - not able to connect to live stream

### DISCUSSION THREAD

**Discussion by wegunterjrFIrefly** [foxglove/community#69] [Q&A]
I have ROS2 Galactic running on another machine (Ubuntu, etc)
FoxGlove Installed - Windows 10
Unable to connect to live
I confirmed ROS_DOMAIN_ID and I am on the same network - but no live topics will show.
I can open a recorded bag (with some topic issues.
Is there something else I am supposed to install on Windows to help with this? Or are all the parts needed in the install?

**Comment by jtbandes** [comment]
Hi, sorry for the delayed response on this issue! We've converted it to a Discussion because it's not a bug report or feature request.

In general, no additional installation should be needed for this to work, so it's likely an issue with network configuration. Can you share more information about your setup? For example is your Windows computer able to access your Ubuntu machine? (by hostname, IP, or what?) Are there any warning or error messages in the desktop app's developer console (ctrl+shift+i)? Can you try using a [Rosbridge connection](https://foxglove.dev/docs/studio/connection/rosbridge) (which has simpler networking requirements – just a single websocket connection) and see if that works any better?

To get more real-time help you can also [join us in Slack](https://foxglove.dev/slack).
